src/main.js

```diff
@@ -21,9 +19,11 @@ const urlParams = new URLSearchParams(window.location.search)
 const showExperimental =
   urlParams.has(SHOW_EXPERIMENTAL_QUERY_PARAM) ? JSON.parse(urlParams.get(SHOW_EXPERIMENTAL_QUERY_PARAM)) : false
 const gitBranch = urlParams.has(BRANCH_QUERY_PARAM) ? urlParams.get(BRANCH_QUERY_PARAM) : 'master'
+const rawGithubContentUrl = `https://raw.githubusercontent.com/transportapi/usage-examples/${gitBranch}/src/examples/`
+const exampleSourceGithubUrl = `https://github.com/transportapi/usage-examples/tree/${gitBranch}/src/examples/`
 
 // For local development load the files from a relative path
-const examplesSourceFilesRootUrl = urlParams.has(LOCAL_DEVELOPMENT_QUERY_PARAM) ? './examples/' : RAW_GITHUB_CONTENT_URL
+const examplesSourceFilesRootUrl = urlParams.has(LOCAL_DEVELOPMENT_QUERY_PARAM) ? './examples/' : rawGithubContentUrl
```

Comment:
We should be consistent with naming. There is `eslint` I set up for this project, probably with a lot of offences and it probably has an opinion on naming. Would be good to clean up and make it part of CI runs

Reply:
Eslint is running in CI. I also have configured it for my IDE so I see live when I violate a rule. The snake case is for constants while the camel case is for variables.

Reply:
What is a constant and what is a variable? I see `const` here :)

Reply:
Both are `const`, yes, they don't change once assigned.
By 'constant' I meant a hard-coded value (url, etc.). The camel case ones are known at runtime.
I remember when I was naming them I followed this convention: https://javascript.info/variables#uppercase-constants
because I couldn't find anything about it in the other style guides.
I can follow any other convention, no problems. Maybe use a camel case for everything?